### PR TITLE
Shawn mock drilldown

### PR DESCRIPTION
**Pull request by shawnkchan**
### Description

Adds i18n wrappers

### Issues Resolved

Partial fix for issue #2 

## Screenshot

NA

## Testing the changes

NA

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by willie-hung** [comment]
Hi @shawnkchan, can u also wrap these lines up. Thanks.
https://github.com/Willie-The-Lord/OpenSearch-Dashboards/blob/fc0915fc8ef20f1ea14b8743d647d59d182c13c3/src/plugins/vis_type_drilldown/public/components/card_form.tsx#L88-L90

**Comment by willie-hung** [comment]
In addition, according to [this information](https://stackoverflow.com/questions/12894169/what-is-the-html-for-attribute-in-label), should we associate the `htmlFor` attribute to the `id` of the input text field?

File: src/plugins/vis_type_drilldown/public/components/card_form.tsx [hunk 1]
```diff
@@ -107,7 +126,13 @@ const CardForm = ({
           <EuiFlexItem>
             <EuiTitle size="xs">
               <h2>
-                <label htmlFor="drilldownVisInput">Select a Destination</label>
+                <label
+                  htmlFor={i18n.translate('drilldownVisInput', {
+                    defaultMessage: 'Drilldown Visualisation Input',
```
Comment (willie-hung) [inline src/plugins/vis_type_drilldown/public/components/card_form.tsx]:
```suggestion
                    defaultMessage: 'Drilldown Visualisation Destination',
```

File: src/plugins/vis_type_drilldown/public/components/card_form.tsx [hunk 2]
```diff
@@ -86,7 +99,13 @@ const CardForm = ({
           <EuiFlexItem>
             <EuiTitle size="xs">
               <h2>
-                <label htmlFor="drilldownVisInput">Url</label>
+                <label
+                  htmlFor={i18n.translate('drilldownVisInput', {
+                    defaultMessage: 'Drilldown Visualisation Input',
```
Comment (willie-hung) [inline src/plugins/vis_type_drilldown/public/components/card_form.tsx]:
```suggestion
                    defaultMessage: 'Drilldown Visualisation Url',
```

File: src/plugins/vis_type_drilldown/public/components/card_form.tsx [hunk 3]
```diff
@@ -65,7 +72,13 @@ const CardForm = ({
           <EuiFlexItem>
             <EuiTitle size="xs">
               <h2>
-                <label htmlFor="drilldownVisInput">Description</label>
+                <label
+                  htmlFor={i18n.translate('drilldownVisInput', {
+                    defaultMessage: 'EG',
```
Comment (willie-hung) [inline src/plugins/vis_type_drilldown/public/components/card_form.tsx]:
```suggestion
                    defaultMessage: 'Drilldown Visualisation Description',
```

Comment (shawnkchan) [reply src/plugins/vis_type_drilldown/public/components/card_form.tsx]: oh whoops sorry i was messing with some things and forgot to change that

**Review by willie-hung — CHANGES_REQUESTED** [review]
Hi @shawnkchan, please rename as is for now, we can check back if it's necessary later

File: src/plugins/vis_type_drilldown/public/components/card_form.tsx [hunk 4]
```diff
@@ -45,7 +46,13 @@ const CardForm = ({
           <EuiFlexItem>
             <EuiTitle size="xs">
               <h2>
-                <label htmlFor="drilldownVisInput">Card Name</label>
+                <label
+                  htmlFor={i18n.translate('drilldownVisInput', {
+                    defaultMessage: 'Card Name',
```
Comment (willie-hung) [inline src/plugins/vis_type_drilldown/public/components/card_form.tsx]:
```suggestion
                    defaultMessage: 'Drilldown Visualisation CardName',
```

**Pull request merged**